Backend/dev/migrations/dynamic-offer-driver-app/0417-added-associatited-for-fleet.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE atlas_driver_offer_bpp.fleet_driver_association ADD COLUMN associated_on TIMESTAMP WITH TIME ZONE;
```

Comment:
We may need to backfill older data.

Reply:
yes @AarushiSharma05 is working on that queries.

Reply:
Yes @AarushiSharma05  is working on this.